index.html

```diff
@@ -1,16 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
-
-<body>
-    <!-- Add your markup here -->
-</body>
-
-</html>
\ No newline at end of file
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./CSS/common.css" />
+    <link rel="stylesheet" href="./CSS/header.css" />
+    <link rel="stylesheet" href="./CSS/main.css" />
+    <link rel="stylesheet" href="./CSS/footer.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo_div">
+        <img src="./img/happy-birthday-cake-icon-18.png" alt="cake logo" />
+      </div>
+      <h1>Specially designed birthday cakes to keep you smiling!</h1>
+      <div class="burger_btn">
+        <div class="burger_item"></div>
+        <div class="burger_item"></div>
+        <div class="burger_item"></div>
+      </div>
+    </header>
+    <main class="main">
+      <nav class="nav">
+        <ul>
+          <li><a href="#Home">Home</a></li>
+          <li><a href="#Cakes">Cakes</a></li>
+          <li><a href="#Shop">Shop</a></li>
+          <li><a href="#About">About</a></li>
+          <li><a href="#Gallary">Gallary</a></li>
+        </ul>
+      </nav>
+      <article class="hero">
+        <section class="hero_text">
+          <h1>Welcome To Sugarholics</h1>
+          <p>
+            Celebrate your special day with our wide range of personalised
+            Birthday cakes. Handcrafted fresh and delivered same day across
+            London. Attractive cake topper candles and balloon accessories
+            available.
+          </p>
+        </section>
+        <section class="hero_img">
+          <img src="./img/1555882.jpeg" alt="A picture of a birthday cake" />
```

Comment:
* Consider to give file a recognisable name to improve code readability.  
* Remember to provide `alt` for support accessibility